corehq/apps/app_manager/migrations/0018_migrate_case_search_labels.py

```diff
@@ -18,7 +10,5 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.RunPython(_migrate_case_search_labels,
-                             reverse_code=migrations.RunPython.noop,
-                             elidable=True),
+        block_upgrade_for_removed_migration("9cc207180e9ebde1c06a98f5e3da1c29b9d37dee"),
```

Comment:
Should this use `prompt_for_historical_migration`? It looks like that and `block_upgrade_for_removed_migration` are quite similar, possibly redundant.

Reply:
https://github.com/dimagi/commcare-hq/pull/31769